python/cudf/cudf/core/indexed_frame.py

```diff
@@ -1324,6 +1324,8 @@ def _drop_na_rows(
         thresh: int, optional
```

Comment:
The spacing is inconsistent here with other parameters.
```suggestion
        thresh : int, optional
```